udata/core/search/commands.py

```diff
@@ -80,11 +76,15 @@ def init(name=None, delete=False, force=False):
         else:
             exit(-1)
 
+    index = Index(index_name, using=es.client)
+    for adapter_class in adapter_catalog.values():
+        index.doc_type(adapter_class)
+    index.create()
+
     adapters = adapter_catalog.items()
     # Ensure that adapters are indexed in the same order.
     adapters.sort()
-    for model, adapter_class in adapters:
-        adapter_class.init(using=es.client, index=es.index_name)
+    for model, adapter_class in reversed(adapters):
```

Comment:
If we keep the `reversed` and that may be a good thing to init geozones at last, we have to document that line.